specifyweb/frontend/js_src/lib/components/QueryBuilder/QueryLineTools.tsx

```diff
@@ -50,9 +54,11 @@ export function QueryLineTools({
       <Button.Small
         aria-label={queryText.showButtonDescription()}
         aria-pressed={field.isDisplay}
-        className={`${className.ariaHandled} ${
-          isFieldComplete ? '' : 'invisible'
-        }`}
+        className={`
+         ${className.ariaHandled} ${isFieldComplete ? '' : 'invisible'} ${
```

Comment:
you could make this quite easier to read by indenting it better:

```ts
    className={`
      ${className.ariaHandled}
      ${isFieldComplete ? '' : 'invisible'}
      ${isBasic ? 'h-full' : ''}
    `}
```

(i.e, a separate line for each ${ block)

isn't this easier to read?